Header header
float32[] data
int32 numberSamples
float32[] wavelengths
float32[] integrationTime
int32[] lampPower
int32[] humidity
int32[] temp